src/configure-webpack/index.js

```diff
@@ -83,6 +83,16 @@ const buildSharedWebpackConfig = (saguiConfig) => {
       })
     ],
 
+    // In wepback 2 builtin node.js modules take precedence over the normal modules
+    // So, if a user creates a `constants` module and tries to require it
+    // with `import 'constants'`, the node.js module will be imported instead
+    //
+    // This configuration disables this behavior
+    // More information: https://github.com/webpack/webpack/issues/4666#issuecomment-292700060
+    node: {
+      constants: false
+    },
+
```

Comment:
You are right, `constants` is really specific case that sagui can handle. Let's keep it.